src/ElectricPokemon.java

```diff
@@ -0,0 +1,114 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+    private final String type = "electric";
```

Comment:
HEt is heel goed om deze variabel `final` te maken. Omdat je het hier definieerd en niet in constructor, mag het zelfs ook `static` zijn. Omdat het een constante is (dus geen variabele), is het conventie om de naam dan in hoofdletters te zetten, dus `TYPE`